L1Trigger/L1TMuon/python/simDigis_cff.py

```diff
@@ -79,8 +79,10 @@
     SimL1TMuon = cms.Sequence(SimL1TMuonCommon + simTwinMuxDigis + simBmtfDigis + simEmtfDigis + simOmtfDigis + simGmtCaloSumDigis + simGmtStage2Digis)
 
     from L1Trigger.ME0Trigger.me0TriggerPseudoDigis_cff import *
+    from L1Trigger.L1TMuon.simDisplacedGmtStage2Digis_cfi import *
```

Comment:
@jimbrooke This PR modifies the Phase-1 global L1Mu trigger so that it writes out a L1T::MuonPhase2 in addition to the L1T::Muon. The purpose of `simDisplacedGmtStage2Digis_cfi` is to calculate the displaced muon pT and set it in L1T::MuonPhase2 using `setPtNoVtx()`. L1T::MuonPhase2 can then be matched to L1Trk -> `prompt L1TrkMu`, or in the other case it can form a `displaced L1TrkMu`. In the trigger meeting you mentioned certain design considerations. Can you elaborate on that?